web-wallet/src/lib/components/__tests__/GasControls.spec.js

```diff
@@ -210,4 +242,8 @@ describe("GasControls", () => {
 
 		expect(limitInput.valueAsNumber).toBe(props.limitLower);
 	});
+
+	it("should dispatch a \"gasSettingsValidity\" event when the price or limit are changed", async () => {
```

Comment:
You forgot this empty test.

Reply:
Do not know how that happened, will get deleted.